### PR TITLE
Added accessibility and licensing criteria categories

### PR DESCRIPTION
**Pull request by davrodgon**
Added qc_accessibility and qc_licensing criteria categories.

### CODE REVIEW

**Comment by orviz** [comment]
thanks @davrodgon for this change. I see that we would need a slightly different approach for these two criteria. As far as I can see it, they won't need the `commands` (or `tox` whatsoever) setting so the following definition:
```
            "oneOf": [
                { "required": ["commands"] },
                { "required": ["tox"] }
            ],
```
should be changed in the `repo_settings` definition. I would say that only the repo is needed. Any thoughts @samuelbernardolip ?

**Comment by davrodgon** [comment]
I only added those two, there some others in the document, but I don't know if we are going to use them.

**Comment by davrodgon** [comment]
> thanks @davrodgon for this change. I see that we would need a slightly different approach for these two criteria. As far as I can see it, they won't need the `commands` (or `tox` whatsoever) setting so the following definition:
> 
> ```
>             "oneOf": [
>                 { "required": ["commands"] },
>                 { "required": ["tox"] }
>             ],
> ```
> 
> should be changed in the `repo_settings` definition. I would say that only the repo is needed. Any thoughts @samuelbernardolip ?

Alternatively we can add the functionality to the docker image. See ci-images PR.

**Comment by orviz** [comment]
> I only added those two, there some others in the document, but I don't know if we are going to use them.

yes of course we will use these criteria. The idea is to have JPL supporting as much criteria from the docs as technically possible. That's why I don't think the Docker image approach is the way to go, the logic to support these criteria should be in the library (JPL).

Since this particular PR is for the schema validator, we need to decide what statements we allow in the `config.yml` for both the qc_accessibility and qc_licensing. In the sqaaas-dev group in Skype, we discussed this. For `qc_licensing` we need:
- Default locations for accessing the license files (e.g. LICENSE* in the root path), but also we should allow to parameterize this path, for those cases where the location is different from the default ones, or in the case that several license files are present (code, documentation)
   ```
   qc_licensing:
     repos:
       foo_repo:
         path: licenses/CODE.md
   ```
   or in case of multiple ones:
   ```
   qc_licensing:
     repos:
       foo_repo:
         path: 
           - licenses/CODE.md
           - licenses/DOCUMENTATION.md
   ```   
- Support for multiple repos:
   ```
   qc_licensing:
     repos:
       - foo_repo
       - bar_repo
   ```
   or if non-standard locations are used for multi-repos:
   ```
   qc_licensing:
     repos:
       foo_repo:
         path: 
           - licenses/CODE.md
           - licenses/DOCUMENTATION.md         
       bar_repo:
         path: licenses/CODE.md    
   ```
- If no repo is defined and `qc_licensing` is there, it will check the code repository that triggered the build. In this case, this implies that only the criterion name is needed:
   ```
   qc_licensing:
   ```

Please @davrodgon and @samuelbernardolip include here any valid statements you think should be supported by the validator.

File: build.gradle [hunk 1]
```diff
@@ -24,18 +24,17 @@ repositories {
...
     // The production code uses the SLF4J logging API at compile time
-    //compile 'org.slf4j:slf4j-api:1.7.21'
+    // compile 'org.slf4j:slf4j-api:1.7.30'
+    runtimeOnly 'org.slf4j:slf4j-simple:1.7.30'
+
 }
 
 mainClassName = 'eu.indigo.jplvalidator.Cli'
-project.ext.organization = 'eoscsynergy'
+project.ext.organization = 'davrodgon'
```
Comment (samuelbernardolip) [inline build.gradle]: @davrodgon Maybe this should be corrected to eoscsynergy for the merge

File: build.gradle [hunk 2]
```diff
@@ -47,7 +46,7 @@ if (System.env.GIT_BRANCH) {
 }
 
 // Google's jib task
-jib.to.image = "eoscsynergy/${project.name}:${version}"
+jib.to.image = "davrodgon/${project.name}:${version}"
```
Comment (samuelbernardolip) [inline build.gradle]: @davrodgon Maybe this should be corrected to eoscsynergy for the merge

**Review by samuelbernardolip — CHANGES_REQUESTED** [review]
For qc_accessibility we need to review the tools to be used[1] and the required additional properties[2] that they should require or not.
[1] https://www.w3.org/WAI/about/projects/wai-tools/third-open-meeting/
[2] https://www.w3.org/TR/WCAG22/

File: build.gradle (hunk 2 above)
Comment (davrodgon) [reply build.gradle]: Yes, sorry, I was using that for testing.

Comment (davrodgon) [reply build.gradle]: Changed

**Comment by orviz** [comment]
Already addressed in https://github.com/EOSC-synergy/jpl-validator/commit/049a4380191b37cb2806b5e3831a3400449847b0

**Pull request closed without merging**